RELEASE-NOTES.txt

```diff
@@ -1,6 +1,7 @@
 4.11.0
 ======
 -   Notes List refreshed to display Pinned state on the right hand side.
+-   iOS 10 Is now the minimum supported OS.
```

Comment:
might be helpful to explain why we're making this change

```
- iOS 10 is now the minimum supported OS: deprecating XYZ, using new ABC interfaces, etc…
```

Reply:
(Just for the record!!) re-replying here. Catch is: the Release Notes are picked up by one of our deployment scripts, and goes directly to the AppStore (prior to being translated).

For that reason, we should keep things as human friendly as possible. Thanks for bringing this up D!